towncrier/newsfragments/1662.feature

```diff
@@ -0,0 +1 @@
+Enable support of all hash types passlib supports. Convert them to the default scheme on first use.
```

Comment:
```suggestion
Enable support of all hash types passlib supports.
```

In accordance with the other suggestion.

Reply:
That's 537bb3c